src/devices/honyar.ts

```diff
@@ -128,6 +129,38 @@ const definitions: Definition[] = [
             await reporting.onOff(endpoint3);
         },
     },
+    {
+        zigbeeModel: ['000a0abb\u0000', 'RH5000_SmartOutlet'],
+        model: 'IHC8223AL',
+        vendor: 'Honyar',
+        description: 'Smart Power Socket 10A (with power monitoring)',
+        extend: [
+            onOff({powerOnBehavior: false}),
+            electricityMeter({cluster: 'honyar', current: {divisor: 1000}, voltage: {divisor: 1}, power: {divisor: 1}}),
+        ],
+        onEvent: async (type, data, device) => {
+            device.skipDefaultResponse = true;
+            const Endpoint = device.getEndpoint(1);
+            if (Endpoint == null) {
+                return;
+            }
+            if (type === 'stop') {
+                clearInterval(globalStore.getValue(device, 'interval'));
+                globalStore.clearValue(device, 'interval');
+            } else if (!globalStore.hasValue(device, 'interval')) {
+                const interval = setInterval(async () => {
+                    try {
+                        await Endpoint.read('haElectricalMeasurement', ['activePower', 'rmsCurrent', 'rmsVoltage']);
+                        await Endpoint.read('seMetering', ['currentSummDelivered']);
+                        await Endpoint.read('genOnOff', ['onOff']);
+                    } catch (error) {
+                        // Do nothing
+                    }
+                }, 5*1000);// Every 5 seconds
```

Comment:
Every 5 seconds is too much, default it to 60 seconds and make it configurable (search for `measurement_poll_interval` in this repo for examples)